docs/source/using/managing-minishift.adoc

```diff
@@ -322,3 +322,52 @@ Enables provisioning the OpenShift link:https://docs.openshift.org/latest/archit
 
 `extra-clusterup-flags`::
 Enables passing flags that are not directly exposed in the Minishift CLI directly to `oc cluster up`.
+
+
+[[hyperv-static-ip]]
+=== Assign IP address to Hyper-V (experimental)
+
+Since the Internal Virtual Switch for Hyper-V does not provide a DHCP offer option, an IP address needs to be provided in a different way.
+On Hyper-V a functionality is provided to assign an IP address on startup using the Data Exchange Service.
+
+[IMPORTANT]
+====
+-  This only works with the CentOS/RHEL based image in combination with Hyper-V
+-  DHCP is still enabled for fallback. Since this will timeout on an internal virtual switch it slightly increases the startup time.
+====
+
+To make this work you need to create a Virtual Switch using NAT
+
+- link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/setup-nat-network[Set up a NAT network]
+
+
+[NOTE]
+====
+WinNAT is limited to one NAT network per host. For more details about capabilities, and limitations, please see the link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/setup-nat-network[WinNAT capabilities and limitations blog]
+====
+
+The following command will attempt to assign an IP address for use on the Virtual Switch 'Internal'.
+
+----
+PS> $env:MINISHIFT_ENABLE_EXPERIMENTAL="y"
+PS> $env:HYPERV_VIRTUAL_SWITCH="Internal"
+PS> minishift.exe start `
+  --iso-url centos `
+  --network-ipaddress 192.168.1.10 `
+  --network-gateway 192.168.1.1 `
+  --network-nameserver 8.8.8.8
+----
+
+
+If you want to use the 'DockerNAT' network, the following commands are needed to setup the correct NAT networking and assigning an IP in the range expected.
```

Comment:
This is the case for using internal virtual switch or DockerNAT which is internal right?